tests/IntegrationTest.php

```diff
@@ -34,7 +34,7 @@ public function testPingCtorWithExplicitUnixUrlSendsRequestToGivenUnixSocket()
         $client = new Client(null, 'unix://' . $path);
         $client->ping();
 
-        $value = \Clue\React\Block\await($deferred->promise(), Loop::get(), 1.0);
+        $value = \React\Async\await(\React\Promise\Timer\timeout($deferred->promise(),1.0));
```

Comment:
It looks like the `Loop` import may no longer be needed (here and elsewhere), can you take a look at this? :+1:

Also, we don't currently have an explicit dependency for the `timeout()` function, should we add this to `require-dev` or perhaps we don't really need this here? What do you think?

Reply:
The test are fine without it so I think we can get rid of it then 👍🏼

Reply:
Removing works for me, the test would run forever if the promise wouldn't resolve.